### PR TITLE
Document channel config

### PR DESCRIPTION
**Pull request by gabrielhase**
This PR does 2 big things:
- completely document the channel-config (and call it project-config)
- fix the evaluation guide (at least everything I found)

### CODE REVIEW

File: reference-docs/project-config/directives/link.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Example
+
+```javascript
+module.exports = {
+    "label": "Web Teaser",
+    "iconUrl": `https://livingdocsio.github.io/livingdocs-design-assets/docs/icons/component-icons/icon_component_teaser_with_image.svg`,
+    "directives": [{
+        "name": "link",
+        "type": "link",
+        "prefill": {
```
Comment (DaRaFF) [inline reference-docs/project-config/directives/link.md:14]: I guess prefill is optional right? So you should mention that the prefill can be left empty.

File: reference-docs/project-config/settings.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+Webhooks allow you to receive notifications for events from within Livingdocs and react to them. We advise you to use the UI under "Project Setup" to define your webhooks.
+
+For details on the parameters see our [Webhooks Guide](../server-configuration/webhooks.md).
+
+## Copy Configuration
+
+The copy configuration allows you to define the mapping of what content-types can be copied to what other content-types and how the metadata is mapped. A common example is the copy of a web article to the print article.
+
+It's an array of copy defintions. Each definition first defines the `source`, i.e. the content-type that is copied (e.g. web article) and then the possible targets (e.g. print article) and which metadata fields should be mapped from source to target.
```
Comment (DaRaFF) [inline reference-docs/project-config/settings.md:167]: typo defintions

File: walkthroughs/getting_started.md [hunk 3]
```diff
@@ -40,16 +40,19 @@ The next step is to create your own metadata field. In the server, navigate into
...
-[use webhooks](../reference-docs/server-configuration/hooks.md)
+[use server hooks](../reference-docs/server-configuration/hooks.md)
+or
+[use webhooks](../reference-docs/server-configuration/webhooks.md)
 
 ## Import
 You can import documents from your old application. There is a special API where you can create documents programmatically. So you have to transform the old format to the livingdocs format and create an article.
 
-[import api](../reference-docs/server-api/import_api.md)
+[import api](https://edit.livingdocs.io/public-api) (under "Import")
```
Comment (DaRaFF) [inline walkthroughs/getting_started.md:50]: We should add an anchor for every section in the public api

File: README.md [hunk 4]
```diff
@@ -1,24 +1,17 @@
...
 
-This is the technical documentation for the Livingdocs CMS. We address 3 larger audiences here:
-- designers
-- integrators
-- core developers
+This is the technical documentation for the "Livingdocs CMS - Enterprise Edition". As a starting point we recommend the Evaluation Guide and then moving on from there.
+Make sure that you have the Livingdocs Boilerplate projects cloned locally as the documentation referes to them in many places
 
-## Livingdocs as a service
+Note: Enterprise means you have access to our source code. If you are a service customer, refer to our [Service Documentation](https://upfrontio.gitbook.io/livingdocs-service/).
```
[the remark this answers is not in the document]
Comment (gabrielhase) [reply README.md:6]: For the service documentation I was thinking about `developers.livingdocs.io`
For the enterprise docs I don't have a really strong opinion, but just `enterprise.livingdocs.io` is misleading if it doesn't mention docs

File: reference-docs/project-config/content_types.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+We plan to allow to move all layout options which are currently defined in the design to the `contentType` configuration. The current situation is an intermediary step to limit the required changes when upgrading to the latest version.
+
+## Example
+
+```javascript
+{
+  handle: 'gallery',
+  documentType: 'article', // either 'article', 'page' or 'data-record'
+  isAuthor: false, // only true if this content-type represents an author, must be 'data-record'
```
[the remark this answers is not in the document]
Comment (gabrielhase) [reply reference-docs/project-config/content_types.md:19]: you can prefill authors: https://docs.livingdocs.io/general-howtos/prefill-author
(but I don't think anyone is using this, it was an idea by DuMont but then they never did it)

File: reference-docs/project-config/content_types.md [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+  },
+
+  // With 'documentCreationDisabled: true', you can't create documents with this content-type
+  documentCreationDisabled: true, // default false
+  viewAfterDocumentCreation: 'publish' // either 'edit' or 'publish', default is 'edit'
+}
+```
+
+## Default Content
+*has UI support*
```
[the remark this answers is not in the document]
Comment (gabrielhase) [reply reference-docs/project-config/content_types.md:104]: I wanted to have it italic. And yes, I was thinking about icons too, but emojis won't work with gitbook. Maybe I can bother Sven with it at some point

File: reference-docs/project-config/content_types.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+
+NOTE: the default content only gets applied when a document is created manually, i.e. in the editor. If you create documents programmatically (e.g. import), you need to define the content yourself.
+
+The default content is simply a (valid) Livingdocs JSON applying to your design. We advise you to use the UI in the "Project Setup" for adapting the default content.
+
+One useful property on components within the default content is the `position: fixed` property. It allows to fix a component in place, useful, e.g. for headers. [Read more](../editor-configuration/editing-features.md#pin-components)
+
+## Editor Wrapper
+
+Defines an HTML string that wraps the document when used in the editor, e.g. for global styling. Use the class `doc-section` where you want the content to be inserted.
```
[the remark this answers is not in the document]
Comment (gabrielhase) [reply reference-docs/project-config/content_types.md:116]: I wouldn't know of any :)

File: reference-docs/project-config/content_types.md [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+
+Apart from the general settings (`renderSettings`) you define an entry for each teaser (`teasers`) giving it the Livingdocs component (from the design) that should be used for rendering as well as a mapping of metadata values to component directives. See our guide on [teaser previews](../server-configuration/teaser-preview-config.md) for more details.
+
+## Desk-Net (has UI support)
+
+We advise you to setup the Desk-Net configuration from the "Project Setup" entry of the same name. There is also a [video](https://vimeo.com/368750546) explaining the setup.
+
+See our [Desk-Net plugin guide](../server-configuration/desknet-integration.md) for comprehensive infos (custom code parts require enterprise version).
+
+## Renditions (enterprise only)
```
[the remark this answers is not in the document]
Comment (gabrielhase) [reply reference-docs/project-config/content_types.md:274]: why? it's only available in the enterprise model

File: reference-docs/project-config/content_types.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (peyerluk) [reply reference-docs/project-config/content_types.md:104]: fyi: you can use utf8 emojis.

File: reference-docs/project-config/content_types.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (peyerluk) [reply reference-docs/project-config/content_types.md:116]: `<div class="doc-section customer-editor-wrapper-styles"></div>`
-> In the editor wrapper you can e.g. define a nice padding around the content that might be different than your default for a nicer editing experience. Or you can overwrite individual styles if you want to make any part of your design look different in the Livingdocs Editor.

File: reference-docs/project-config/directives/include.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  `
+}
+```
+
+## Config Options
+
+### Parameters
+
+- `config`: object, allows you to pass static configuration to your third-party microservice. In the example above we pass a language as a fixed config.
+- `defaultParams`: object, allows you to define defaults for the dynamic params of an include. Dynamic params are set by the user over a UI in the editor and are defined in the [Settings for the include](../settings.md#includes).
```
[the remark this answers is not in the document]
Comment (peyerluk) [reply reference-docs/project-config/directives/include.md:36]: fyi: locally some markdown plugins support links in their previews.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (peyerluk) [reply README.md:6]: developers.google.com
developer.twitter.com

File: walkthroughs/getting_started.md (hunk 3 above)
Comment (gabrielhase) [reply walkthroughs/getting_started.md:50]: I already told @peyerluk  :)

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (peyerluk) [reply README.md:6]: maybe:
`developer.livingdocs.io` -> service
`developer.livingdocs.io/entreprise` -> enterprise
`docs.livingdocs.io` -> redirect to `developer.livingdocs.io`
`developers.livingdocs.io` -> redirect to `developer.livingdocs.io`

[the remark this answers is not in the document]
Comment (gabrielhase) [reply README.md:6]: As I said :)

File: reference-docs/project-config/content_types.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (DaRaFF) [reply reference-docs/project-config/content_types.md]: It's only about consistency not that the info is wrong.

File: reference-docs/project-config/README.md [hunk 10]
```diff
@@ -1,24 +1,11 @@
-# Channel Configuration
+# Project Config
 
-## Summary
+The project config affects all configurable settings of a Livingdocs project, e.g. the content types or the design. There can be several project configs within a Livingdocs installation, one for each project.
 
-The channel configuration allows you to:
+The project config has wide UI support. Basically the whole section "Project Setup" in the Livingdocs Main Menu allows setting parts of the Project config via UI options. The parts that have UI support are marked in the documentation.
```
[the remark this answers is not in the document]
Comment (benib) [reply reference-docs/project-config/README.md]: I would argue:
A Menu is a thing that let you do something
A Navigation is a thing that let's you go somewhere

**Pull request merged**